nuxeo/templates/_warnings.tpl

```diff
@@ -0,0 +1,91 @@
+{{/* vim: set filetype=mustache: */}}
+
+{{- define "nuxeo.warnings.message" -}}
+{{- $messages := list -}}
+{{- $messages := append $messages (include "nuxeo.warnings.message.binary.cloudProvider" .) -}}
+{{- $messages := append $messages (include "nuxeo.warnings.message.binary.pvc" .) -}}
+{{- $messages := append $messages (include "nuxeo.warnings.message.ingress.sticky-session" .) -}}
+{{- $messages := append $messages (include "nuxeo.warnings.message.log.pvc" .) -}}
+{{- $messages := append $messages (include "nuxeo.warnings.message.rolling-tag" .) -}}
+{{- $messages := without $messages "" -}}
+{{- $message := join "\n\n" $messages -}}
+
+{{- if $message -}}
+{{- printf "\n" }}
+{{- printf "\n" }}
+-------------------------------------------------------------------------------
+WARNING
+{{ $message }}
+{{- printf "\n" }}
+-------------------------------------------------------------------------------
+{{- end -}}
+{{- end -}}
+
+{{- define "nuxeo.warnings.message.binary.cloudProvider" -}}
+{{- if not (include "nuxeo.binary.cloudProvider.enabled" .) -}}
+  {{- if not (.Values.persistentVolumeStorage.enabled) }}
+  By not enabling any cloud provider for binary storage, binaries will be
+  stored in an emptyDir volume, thus not surviving a pod restart.
+  {{- end }}
+  Using a cloud provider for binary storage is preferred for production.
+  You can enable one by setting either:
+    googleCloudStorage.enabled=true or amazonS3.enabled=true.
+{{- end -}}
+{{- end -}}
+
+{{- define "nuxeo.warnings.message.binary.pvc" -}}
+{{- if .Values.persistentVolumeStorage.enabled -}}
+  {{- if include "nuxeo.binary.pvc.has-many" . }}
+  Enabling a persistent volume for binary storage with the ReadWriteMany
+  access mode is not supported by the Nuxeo Helm Chart.
+  Additional configuration might need to be added,
+  see https://doc.nuxeo.com/nxdoc/file-storage-architecture/.
+  {{- else }}
+  By enabling a persistent volume for binary storage, since it is mounted
+  with the ReadWriteOnce access mode, the deployment won't be able
+  to scale up in Kubernetes.
+  {{- end }}
+  It is not recommended for production, a cloud provider for binaries is
+  preferred.
+{{- end -}}
+{{- end -}}
+
+{{- define "nuxeo.warnings.message.ingress.sticky-session" -}}
+{{- if and (.Values.ingress.enabled) (include "nuxeo.clustering.enabled" .) }}
+  Nuxeo needs sticky session for cookie to work correctly. Unfortunately
+  this can not be achieved by the Nuxeo Helm Chart as it depends on your
+  Ingress controller, see below some examples:
+  * NGINX - add the annotation below to the ingress:
+      ingress:
+        annotations:
+          nginx.ingress.kubernetes.io/affinity: "cookie"
```

Comment:
You mean handling the NGINX case?

I would rather not add it as it is not a generic annotation, even if it is the common ingress controller, otherwise we open the door to handle every ingress controllers, and I would rather not do that.
For instance we could have support tickets about customer successfully deploying to their staging/prod k8s cluster where NGINX is the ingress controller but customer is having issue on its dev cluster where Traefik is the ingress controller.

Reply:
OK, makes sense!